dolphinscheduler-alert/dolphinscheduler-alert-server/src/main/java/org/apache/dolphinscheduler/alert/AlertPluginManager.java

```diff
@@ -53,25 +52,22 @@
 public final class AlertPluginManager {
     private static final Logger logger = LoggerFactory.getLogger(AlertPluginManager.class);
 
-    private final PluginDao pluginDao;
+    @Autowired
+    private PluginDao pluginDao;
 
     private final Map<Integer, AlertChannel> channelKeyedById = new HashMap<>();
 
     private final PluginParams warningTypeParams = getWarningTypeParams();
 
-    public AlertPluginManager(PluginDao pluginDao) {
-        this.pluginDao = pluginDao;
-    }
-
```

Comment:
This part is a different spring injection method, it is not recommended to modify.

Reply:
> This part is a different spring injection method, it is not recommended to modify.

yes, I know, first, this is constructor injection, and has the same effect as @Autowired, both are byType, secondly, keep the style consistent, such as master/worker. What do you think?

Reply:
I recommend keeping the original constructor injection and see what other small partners have suggestions.

Reply:
> I recommend keeping the original constructor injection and see what other small partners have suggestions.

Okay,  i agree with you.